docs/examples/ex_init.rst

```diff
@@ -0,0 +1,28 @@
+Loading F\ :sub:`10.7`\
+========================
+
+pysatSpaceWeather uses `pysat <https://github.com/pysat/pysat>`_ to load
+different historical, real-time, and forecasted solar, geomagnetic, and other
+space weather indices.  As specified in the
+`pysat tutorial <https://pysat.readthedocs.io/en/latest/tutorial.html>`_,
+data may be loaded using the following commands.  Historic F\ :sub:`10.7`\ is
+used as an example.
+
+::
+
+
+   import datetime as dt
+   import pysat
+   import pysatSpaceWeather as py_sw
+
+   old_time = dt.datetime(2012, 5, 14)
+   f107 = pysat.Instrument(inst_module=py_sw.instruments.sw_f107, tag='all',
+                           update_files=True)
```

Comment:
This code goes a little outside the code box on RTD.

```suggestion
   f107 = pysat.Instrument(inst_module=py_sw.instruments.sw_f107, 
                           tag='all', update_files=True)
```

Reply:
I think that's a function of your browser config, but I made the change.